File: jambeth/jambeth-cache/src/main/java/de/osthus/ambeth/cache/ISecondLevelCacheManager.java
package de.osthus.ambeth.cache;

public interface ISecondLevelCacheManager
{
	IRootCache selectSecondLevelCache();

	IRootCache selectPrivilegedSecondLevelCache(boolean forceInstantiation);

	IRootCache selectNonPrivilegedSecondLevelCache(boolean forceInstantiation);
}

File: ambeth/Ambeth.Cache/ambeth/cache/ISecondLevelCacheManager.cs
using System;

namespace De.Osthus.Ambeth.Cache
{
    public interface ISecondLevelCacheManager
    {
        IRootCache SelectSecondLevelCache();

        IRootCache SelectPrivilegedSecondLevelCache(bool forceInstantiation);

        IRootCache SelectNonPrivilegedSecondLevelCache(bool forceInstantiation);
    }
}